### ISSUE TITLE
No examples in the doc?

### ISSUE THREAD

**Issue by biohazardxxx** [tadodotcom/grails-quick-search#9]
Hi,

unfortunatly this probably great plugin has a very light documentation. An example how to use it would a lot more people try it out. Please consider to post an example of usage.

Best regards
-Bio

**Comment by philrice** [comment]
i agree , im having to try to work out how to put this in my view to call teh action correctly and I havent even started looking at autocomplete action . No examples at all so Im a bit lost not being a bit of a newbie. I cant find ANY search plugins that work for me so it is quite frustrating and this looked like it might just be the one (actually got it to install without erroring unlike elasticsearch, searchable etc) but i dont know how to actually use it

**Comment by IMCamilo** [comment]
No examples at all so Im a bit lost. I cant find ANY search plugins that work for me so it is quite frustrating and this looked like it might just be the one

**Comment by biohazardxxx** [comment]
I went for elasticsearch. It is aslo a bit of hassle but works fine in the end.

**Comment by IMCamilo** [comment]
U have some example, i need help D:

**Comment by biohazardxxx** [comment]
basically follow the instructions in the plugin [documentation ](http://noamt.github.io/elasticsearch-grails-plugin/docs/index.html) and look at this when you plan for Production:
https://github.com/noamt/elasticsearch-grails-plugin/issues/149#issuecomment-235633518

at config needed:
elasticSearch.cluster.name = 'myCluster'
		elasticSearch.client.mode = 'local'
		elasticSearch.index.store.type = 'simplefs'	

at domain:
	static searchable = {
		only  = ['firstname', 'lastname']
	}	
in controller like [described](http://noamt.github.io/elasticsearch-grails-plugin/docs/index.html#searching). in GSP the usual way.

I suggest to open a stackoverflow question for this since that issue here is not the right place.

**Comment by kuceram** [comment]
Have you seen the documentation for this plugin?
http://tadodotcom.github.io/grails-quick-search/guide/index.html
The plugin is really light weight so there is not much to document, however examples are in documentation, see http://tadodotcom.github.io/grails-quick-search/ref/Services/searchService.html.

It would definitely make sense to have the link in readme file of this project (https://github.com/tadodotcom/grails-quick-search/pull/18).
